plugin/controller/lib/impl/mcp/MCPControllerRegister.ts

```
@@ -0,0 +1,450 @@
+    } else if (MCPControllerRegister.hooks.length > 0) {
+      for (const hook of MCPControllerRegister.hooks) {
+        schema = await hook.schemaLoader?.(controllerMeta, toolMeta);
+        if (schema) {
+          args.push(schema);
+          break;
+        }
+      }
+    }
+    const self = this;
+    const handler = async (...args) => {
+      const eggObj = await self.eggContainerFactory.getOrCreateEggObject(controllerProto, controllerProto.name);
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)